### PR TITLE
Kegerface sql slim

### PR DESCRIPTION
**Pull request by Skemp45**
This Has all of the most current updates as of 1/08/2013 tam Arizona time

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -140,15 +139,24 @@ img {
...
 /* Indivual Row Styles for Beers */
 .BeerWrapper{
 	width:98%;
-	min-height:100px;
+	min-height:95px;
 	height:auto !important;
-	height:100px;
+	height:95px;
 	background-color: rgba(153,153,153,.3);
-	padding:5px 1% 5px 1%;
```
Comment (Skemp45) [inline style.css:150]: this was causing our issues with dashed borders not lining up

File: style.css [hunk 2]
```diff
@@ -140,15 +139,24 @@ img {
...
-	margin-bottom:5px;
```
Comment (Skemp45) [inline style.css:151]: this was causing our issues with dashed borders not lining up

File: style.css [hunk 3]
```diff
@@ -84,7 +84,6 @@ img {
 	width:98%;
 	height:60px;
 	background-color: rgba(153,153,153,.3);
-	padding:1%;
 	margin-bottom:10px;
```
Comment (Skemp45) [inline style.css:87]: this adds the small gap from title and beers that I feel makes a nice separation from the title bar

File: style.css [hunk 4]
```diff
@@ -84,7 +84,6 @@ img {
 	width:98%;
 	height:60px;
 	background-color: rgba(153,153,153,.3);
-	padding:1%;
```
Comment (Skemp45) [inline style.css:87]: this was causing our issues with dashed borders not lining up

**Comment by thadius856** [comment]
Odd. The diffs aren't show the SQL work. Hrm. But I can see it in the raw files.

I wonder if maybe merging branches doesn't automatically adopt all previous contributor edits for attribution, like forking does.

Anyway, thanks for fixing. :+1:

**Pull request merged**